### DISCUSSION TITLE
Does magit have commitizen or similar functionality to normalize commit messages?

### DISCUSSION THREAD

**Discussion by fzcsyglq** [magit/magit#5186] [Support] [answered]
I just started trying magit. I previously used git cz instead of git commit to standardize commit messages. I wonder if I can do this in magit.

**Comment by tarsius** [comment, marked as answer]
Magit has no support for using alternative commit commands. You would have to create your own command. Start by copying `magit-commit-create` and making the necessary adjustments, maybe just replacing "commit" with "cz". Then bind that command in the `magit-commit` menu (see https://magit.vc/manual/transient/Modifying-Existing-Transients.html).
